.github/workflows/build.yaml

```
@@ -1,43 +1,58 @@
-name: build
+name: CI
 
-on: [push, pull_request]
+on:
+  # Trigger on all pushes and PRs
+  push:
+  pull_request:
+
+  # Allow manually running workflow
+  workflow_dispatch:
```

```suggestion
on: [push, pull_request, workflow_dispatch]
```
As the writer of the original workflow I would like to request this change. It's essentially the same, but only uses one line of code instead of 7.

---

A specific PR (which disables the workflow for small commits, such as a README change) uses my current format, so it's not logical to simply use a list after that PR is merged.

---

Damn. I totally forgot. I wrote that PR. I didn't keep merge issues in mind while making this suggestion